src/index.html

```diff
@@ -17,16 +17,32 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search--form search--form__big"
+      data-qa="big"
+    >
+      <label class="search--form__label search--form__label_big">
+        <input
+          type="text"
+          data-qa="keypress"
+          class="search--form__input search--form__input_big"
+          placeholder="Try “Los Angeles“"
+        />
+      </label>
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search--form search--form__small"
+      data-qa="small"
```

Comment:
Keep your attributes correctly formatted. If the HTML-element has long attribute values or the number of attributes is more than 2, start each one, including the first, on the new line with 2-space indentation related to the tag.